apps/console/src/extensions/configs/attribute.ts

```diff
@@ -1,7 +1,7 @@
 /**
- * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

Comment:
Keep the original year.

```suggestion
 * Copyright (c) 2021, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```